crates/sui-json-rpc/src/name_service.rs

```diff
@@ -48,7 +48,7 @@ pub struct Table<K, V> {
 
 #[derive(Debug, Serialize, Deserialize, Clone, Eq, PartialEq)]
 pub struct Domain {
-    labels: Vec<String>,
+    pub labels: Vec<String>,
```

Comment:
I would probably leave this as private and add an accessor to get a "&[String]"

Reply:
Great catch + thanks for the context. (I don't even need access to it, I think I did it when first starting to play with the data).